app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -50,22 +46,11 @@ var RepositoryDatatable = (function(global) {
       return value;
     });
 
-  function changeToViewMode() {
-    currentMode = 'viewMode';
-    // Table specific stuff
-    TABLE.button(0).enable(true);
-    FilePreviewModal.init();
-  }
-
-  function changeToEditMode() {
-    currentMode = 'editMode';
-    // Table specific stuff
-    TABLE.button(0).enable(false);
-  }
-
   // Enable/disable edit button
   function updateButtons() {
```

Comment:
Maybe we can somehow refactor this function? Add general class for element which must be disabled.

Reply:
It won't work here due to complex switch on/off scenarios